fe/fe-core/src/main/java/com/starrocks/transaction/PartitionCommitInfo.java

```diff
@@ -66,6 +67,8 @@ public class PartitionCommitInfo implements Writable {
     @SerializedName(value = "compactionScore")
     private Quantiles compactionScore;
 
+    private BinlogConfig binlogConfig;
```

Comment:
Why put it into `PartitionCommitInfo` ?

Reply:
When the binlog is enabled, the current alter table logic cannot ensure that the binlog on the BE  is also enabled, so that BE will not produce binlog during the publish version, so the latest binlog config needs to be brought to be through partitionCommitInfo